assets/css/custom-props/theme-light.css

```diff
@@ -80,6 +80,7 @@
   --sidebarHover:                var(--black);
   --sidebarScrollbarThumb:       var(--coldGrayLight);
   --sidebarScrollbarTrack:       var(--sidebarBackground);
+  --sidebarStaleVersion:         var(--orangeLight);
```

Comment:
```suggestion
  --sidebarStaleVersion:         var(--orangeDark);
```

Reply:
I though it looks off 😅 
Addressed in 5dd3ff6f